lib/livebook_web/live/file_select_component.ex

```diff
@@ -482,6 +516,39 @@ defmodule LivebookWeb.FileSelectComponent do
     {:noreply, socket}
   end
 
+  def handle_event("validate", %{}, socket) do
+    Enum.any?(socket.assigns.uploads.folder.entries, &(not &1.valid?))
+    {:noreply, socket}
+  end
+
+  def handle_event("add-file", %{}, socket) do
+    consume_uploaded_entries(socket, :folder, fn %{path: file_path}, _entry ->
+      content = File.read!(file_path)
+
+      file_path =
+        FileSystem.File.resolve(
+          socket.assigns.current_dir,
+          Enum.at(socket.assigns.uploads.folder.entries, 0).client_name
```

Comment:
I think this sholuld be `entry.client_name`, since there can be multiple entries.

Reply:
The file upload is to handle a file at time. and `[Enum.at](socket.assigns.uploads.folder.entries, 0).client_name` was to resolve file name at that instance

Reply:
@gitstart I believe you can get the entry from line 525. Rename `_entry` -> `entry`. :)